### PR TITLE
chore: Update Jenkins version to fix merge conflict

### PR DESCRIPTION
**Pull request by rishabh96b**

### CODE REVIEW

**Review by armandgrillet — DISMISSED** [review]

File: addons/jenkins/1.x/jenkins.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: jenkins
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.9.4-1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.9.4-2"
```
Comment (armandgrillet) [inline addons/jenkins/1.x/jenkins.yaml:9]: I'm wondering if this should not be `2.5.4-1` with such a chart version bump 🤔.

Comment (rishabh96b) [reply addons/jenkins/1.x/jenkins.yaml:9]: Yeah, it is like that in the master branch. Doesn't seem right though.

Comment (makkes) [inline addons/jenkins/1.x/jenkins.yaml:9]:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "2.5.4-1"
```
This repo should follow the guidelines in https://github.com/mesosphere/kubernetes-base-addons/blob/master/CONTRIBUTING.md#addon-revisions IMHO.

File: addons/jenkins/1.x/jenkins.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ metadata:
...
     appversion.kubeaddons.mesosphere.io/jenkins: "1.9.4"
```
Comment (makkes) [inline addons/jenkins/1.x/jenkins.yaml:10]:
```suggestion
    appversion.kubeaddons.mesosphere.io/jenkins: "2.5.4"
```

**Review by makkes — CHANGES_REQUESTED** [review]

**Review by armandgrillet — APPROVED** [review]

**Review by makkes — APPROVED** [review]

**Pull request merged**